web/frontend/src/components/slam-2d-render.vue

```diff
@@ -7,42 +7,36 @@ import * as THREE from 'three';
 import { MapControls } from 'three/examples/jsm/controls/MapControls';
 import { PCDLoader } from 'three/examples/jsm/loaders/PCDLoader';
 import type { commonApi } from '@viamrobotics/sdk';
-import { SVGLoader } from 'three/examples/jsm/loaders/SVGLoader';
-import { baseMarkerUrl } from '../lib/base-marker-url';
-import { destMarkerUrl } from '../lib/destination-marker-url';
-
-type SvgOffset = {
-  x: number,
-  y: number,
-  z: number
-}
+import DestMarker from '../lib/destination-marker.txt?raw';
+import BaseMarker from '../lib/base-marker.txt?raw';
+
+let pointsMaterial: THREE.PointsMaterial | undefined;
 
 const backgroundGridColor = 0xCA_CA_CA;
 
 const gridSubparts = ['AxesPos', 'AxesNeg', 'Grid'];
 
-const gridHelperRenderOrder = 997;
-const axesHelperRenderOrder = 998;
-const svgMarkerRenderOrder = 999;
+const svgMarkerRenderOrder = 4;
+const pointsRenderOrder = 3;
+const axesHelperRenderOrder = 2;
+const gridHelperRenderOrder = 1;
 
 const gridHelperScalar = 4;
 const axesHelperSize = 8;
 
-// Note: updating the scale of the destination or base marker requires an offset update
-const baseMarkerScalar = 0.002;
```

Comment:
Why do we no longer need these scalars?

Reply:
before we were at the mercy of how the svgs themselves decided how they were sized. With a texture / sprite the sizing is entirely based on the accompanying geometry (which for the icons currently matches 1m). I think this is much simpler and easy to work with :)